XCUITests/FxScreenGraph.swift

```diff
@@ -791,7 +791,7 @@ extension Navigator where T == FxUserState {
 
     // Open a URL. Will use/re-use the first BrowserTab or NewTabScreen it comes to.
     @available(*, deprecated, message: "use openURL(_ urlString) instead")
-    func openURL(urlString: String) {
+    func openURL(String) {
```

Comment:
How does this work?

Reply:
Right, I should have removed that entirely, will do